packages/stream/README.md

```diff
@@ -73,6 +73,70 @@ which promises settle.
 A stream is consequently a pair of queues that transport iteration results,
 one to send messages forward and another to receive acknowledgements.
 
+An async queue is itself an input and output pair.
+The `put` function makes it an async sink, and the `get` function makes it an
+async spring.
+
+## PubSub
+
+We can use streams to publish and subscribe to topics.
+
+- `makeLatestTopic` is suitable for values or snapshots that are periodically
+  updated.
+  New subscribers will see the latest published value immediately (or wait for
+  the first published value).
+  The topic discards intermediate values, so a slow subscriber will not see all
+  values from a fast publisher.
+
+- `makeChangeTopic` is suitable for deltas, patches, or changes starting at an
+  initial value or snapshot.
+  New subscribers will wait for the next published value.
+  Every subscriber will see every value after they subscribe.
+
+  For example, consider a program that tracks pending requests.
+  Each new subscriber should see all currently pending requests and
+  then each subsuequent request as it arrives.
+  The subscriber side would:
+
+  ```js
+  const requests = new Map();
+  const requestsTopic = makeChangeTopic();
+  async function* followRequests() {
+    const pendingRequests = Array.from(requests.values());
+    const subsequentRequests = requestsTopic.subscribe();
+    yield* pendingRequests;
+    yield* subsequentRequests;
+  }
+  ```
+
+- `makeUpdateTopic` preserves intermediate values and each subscriber sees the
+  last published value.
+
+Consumers do not need to expressly dispose of their subscriber.
+Releasing the subscriber for garbage collection is sufficient.
+
+The `makeTopic` functions returns a kit with `publish` and `subscribe`
+functions.
+The `publish` function returns a writer but provides no back-pressure.
+The `subscribe` function returns a reader and also provides no back-pressure.
+
+```js
+const { publish, subscribe } = makeLatestTopic();
```

Comment:
This example cannot work when Alice calls `makeLatestTopic()`, Bob uses `publisher` and Carol uses `subscriber1`.

It is alright with me if you know this and don't yet want to add the ability to have pubsub across vats or a captp loopback, but please add a TODO.

```suggestion
// TODO: demonstrate usage of topics across separate vats.
const { publish, subscribe } = makeLatestTopic();
```